primitives/rialto/src/lib.rs

```diff
@@ -132,9 +110,10 @@ pub fn derive_account_from_millau_id(id: bp_runtime::SourceAccount<AccountId>) -
 	AccountIdConverter::convert(encoded_id)
 }
 
-/// Get a struct which defines the weight limits and values used during extrinsic execution.
-pub fn runtime_block_weights() -> frame_system::limits::BlockWeights {
-	frame_system::limits::BlockWeights::builder()
+frame_support::parameter_types! {
+	pub BlockLength: limits::BlockLength =
+		limits::BlockLength::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
+	pub BlockWeights: limits::BlockWeights = limits::BlockWeights::builder()
```

Comment:
Not related, to this PR, but probably this builder may be refactored to const methods and we will be able to mark this parameter as const. Imo it'll be better to expose this struct (`BlockWeights` and `BlockLength`) as const from this crate.